### ISSUE TITLE
In functions that return values, `use` "printf -v output_var_name $value" instead of "eval" to return values

### ISSUE THREAD

**Issue by gsketefian** [NOAA-EMC/regional_workflow#652]
## Description
In functions that return values, currently 

`eval ${var_name}=${var_value}`

is used to set variables to values.  Instead of this, use

`printf -v "var_name" ${var_value}`

to reduce use of the `eval` utility.

## Solution
Use `printf -v` instead of `eval`, as described above.

**Comment by JeffBeck-NOAA** [comment]
@gsketefian, do you have a specific list of files that need to be updated?  Did Daniel's PR moving a number of shell-based scripts to Python resolve this issue?

**Comment by gsketefian** [comment]
I don't think Daniel removed the shell scripts yet.  I guess when he does, this will become moot.

**Comment by JeffBeck-NOAA** [comment]
Right, we'll need to wait until the shell scripts are actually removed in favor of the Python scripts.
